v03/governance/01-raise-validators-number/proposal.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Looks good. 
Self check list -
-> Have you tried this in your local machine. 
-> Have you already tested the inactive -> active validator transition. And active->inactive transition.  
-> If you directly do this proposals you won't be able to test this type of use cases.

Reply:
I highly recommend to not do this proposal without doing this test cases.